store/server/features/approver/mutation.ts

```
@@ -177,6 +184,13 @@
           description: 'Approver updated successfully ',
         });
       },
+      onError: (error: any) => {
+        const errorMessage = error?.response?.data?.message || 'Something went wrong';
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)